_opensearch/query-dsl/term.md

```diff
@@ -329,9 +329,9 @@ GET products/_search
 }
 ```
 
-Specify relative dates by using basic math expressions.
+Specify relative dates by using [date math]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/date/#date-math).
 
-To subtract 1 year and 1 day from the specified date:
+To subtract one year and one day from the specified date:
```

Comment:
Can we make this a complete sentence? Use numerals for all measurement-based references, including time.